### PR TITLE
docs | tasks | configure-pod-container | Static Pods

### PR DESCRIPTION
**Pull request by icheikhrouhou**
Translate Static pods FR

### CODE REVIEW

File: content/fr/docs/tasks/configure-pod-container/static-pod.md [hunk 1]
```diff
@@ -0,0 +1,220 @@
...
+{{% capture overview %}}
+
+
+Les *pods statiques* sont gérés directement par le démon de kubelet sur un noeud spécifique,
+sans les observant par l'{{< glossary_tooltip text="API server" term_id="kube-apiserver" >}}
+Contrairement aux pods qui sont gérés par le control plane (par exemple, un {{< glossary_tooltip text="Deployment" term_id="deployment" >}}), le kubelet surveille chaque pod statique (et le redémarre s'il plante).
+
+Les pods statiques sont toujours liés à un {{< glossary_tooltip term_id="kubelet" >}} sur un noeud spécifique.
+
+Le kubelet crée automatiquement un {{< glossary_tooltip text="Pod mirror" term_id="mirror-pod" >}} sur le API Server de Kubernetes pour chaque Pod statique.
```
Comment (sftim) [inline content/fr/docs/tasks/configure-pod-container/static-pod.md:16]:
```suggestion
Le kubelet crée automatiquement un {{< glossary_tooltip text="Pod mirroir" term_id="mirror-pod" >}} sur le API Server de Kubernetes pour chaque Pod statique.
```
?

File: content/fr/docs/tasks/configure-pod-container/static-pod.md [hunk 2]
```diff
@@ -0,0 +1,220 @@
...
+Cela signifie que les pods fonctionnant sur un noeud sont visibles sur le API Server, mais ne peuvent pas être contrôlées à partir de là.
+
+{{< note >}}
+Si vous utilisez un clustered Kubernetes et des pods statiques pour faire tourner un pod sur chaque noeud, vous devriez probablement utiliser un {{< glossary_tooltip text="DaemonSet" term_id="daemonset" >}} à la place.
```
Comment (sftim) [inline content/fr/docs/tasks/configure-pod-container/static-pod.md:20]: “un clustered Kubernetes” ou “un cluster Kubernetes”?

Aussi, peut-être: “vous devriez envisager de utiliser”?

File: content/fr/docs/tasks/configure-pod-container/static-pod.md [hunk 3]
```diff
@@ -0,0 +1,220 @@
+---
+title: Créer des Pods statiques
+weight: 170
+content_template: templates/task
+---
+
+{{% capture overview %}}
+
+
+Les *pods statiques* sont gérés directement par le démon de kubelet sur un noeud spécifique,
```
Comment (mhamdisemah) [inline content/fr/docs/tasks/configure-pod-container/static-pod.md:10]:
```suggestion
Les *pods statiques* sont gérés directement par le deamon de kubelet sur un noeud spécifique,
```
je pense deamon c'est mieux que démon

File: content/fr/docs/tasks/configure-pod-container/static-pod.md [hunk 4]
```diff
@@ -0,0 +1,220 @@
...
+sans les observant par l'{{< glossary_tooltip text="API server" term_id="kube-apiserver" >}}
```
Comment (mhamdisemah) [inline content/fr/docs/tasks/configure-pod-container/static-pod.md:11]:
```suggestion
sans les observer par l'{{< glossary_tooltip text="API server" term_id="kube-apiserver" >}}
```

File: content/fr/docs/tasks/configure-pod-container/static-pod.md [hunk 5]
```diff
@@ -0,0 +1,220 @@
...
+Contrairement aux pods qui sont gérés par le control plane (par exemple, un {{< glossary_tooltip text="Deployment" term_id="deployment" >}}), le kubelet surveille chaque pod statique (et le redémarre s'il plante).
+
+Les pods statiques sont toujours liés à un {{< glossary_tooltip term_id="kubelet" >}} sur un noeud spécifique.
+
+Le kubelet crée automatiquement un {{< glossary_tooltip text="Pod mirror" term_id="mirror-pod" >}} sur le API Server de Kubernetes pour chaque Pod statique.
+Cela signifie que les pods fonctionnant sur un noeud sont visibles sur le API Server, mais ne peuvent pas être contrôlées à partir de là.
```
Comment (mhamdisemah) [inline content/fr/docs/tasks/configure-pod-container/static-pod.md:17]:
```suggestion
Cela signifie que les pods fonctionnant sur un noeud sont visibles sur l'API Server, mais ne peuvent pas être contrôlées à partir de là.
```

File: content/fr/docs/tasks/configure-pod-container/static-pod.md [hunk 6]
```diff
@@ -0,0 +1,220 @@
...
+```
+
+Le résultat obtenu ressemble à cela :
+
+```
+CONTAINER ID IMAGE         COMMAND  CREATED        STATUS         PORTS     NAMES
+f6d05272b57e nginx:latest  "nginx"  8 minutes ago  Up 8 minutes             k8s_web.6f802af4_static-web-fk-node1_default_67e24ed9466ba55986d120c867395f3c_378e5f3c
+```
+
+Vous pouvez voir le pod miroir sur le API Server :
```
Comment (mhamdisemah) [inline content/fr/docs/tasks/configure-pod-container/static-pod.md:149]:
```suggestion
Vous pouvez voir le pod miroir sur l'API Server :
```

File: content/fr/docs/tasks/configure-pod-container/static-pod.md [hunk 7]
```diff
@@ -0,0 +1,220 @@
...
+```shell
+kubectl get pods
+```
+```
+NAME                       READY     STATUS    RESTARTS   AGE
+static-web-my-node1        1/1       Running   0          2m
+```
+
+{{< note >}}
+Vérifiez que le kubelet a la permission de créer le pod miroir dans le API Server. Sinon, la demande de création est rejetée par le API Server. Voir
```
Comment (mhamdisemah) [inline content/fr/docs/tasks/configure-pod-container/static-pod.md:160]:
```suggestion
Vérifiez que le kubelet a la permission de créer le pod miroir dans l'API Server. Sinon, la demande de création est rejetée par l'API Server. Voir
```

File: content/fr/docs/tasks/configure-pod-container/static-pod.md (hunk 1 above)
Comment (mhamdisemah) [inline content/fr/docs/tasks/configure-pod-container/static-pod.md:16]:
```suggestion
Le kubelet crée automatiquement un {{< glossary_tooltip text="Pod mirror" term_id="mirror-pod" >}} sur l'API Server de Kubernetes pour chaque Pod statique.
```

File: content/fr/docs/tasks/configure-pod-container/static-pod.md (hunk 3 above)
Comment (sftim) [reply content/fr/docs/tasks/configure-pod-container/static-pod.md:10]: daemon, non?

Comment (mhamdisemah) [reply content/fr/docs/tasks/configure-pod-container/static-pod.md:10]: oui daemon ,j'ai corrigé

**Comment by sftim** [comment]
/hold

With 2700 commits, this pull request does not look right.

**Comment by raelga** [comment]
Hello @icheikhrouhou, looks like you PR needs a rebase!

Feel free to reach us in slack to help you with getting your PR ready to merge!

**Comment by inductor** [comment]
@raelga  @icheikhrouhou Could we get unrelated language lables reoved anytime soon? 

Thanks :pray:

**Comment by raelga** [comment]
> @raelga @icheikhrouhou Could we get unrelated language lables reoved anytime soon?

You can remove labels with:

/remove area-blog
/remove language-de
/remove language-en
/remove language-es
/remove language-id
/remove language-it
/remove language-ja
/remove language-ko
/remove language-pl
/remove language-pt
/remove language-ru
/remove language-uk
/remove language-vi
/remove language-zh

I kept only the `language-fr` as originally intended.

**Comment by inductor** [comment]
@raelga Yeah but we need rebase to solve the actual problem. As long as the diff includes other commits we cannot ignore this

**Comment by raelga** [comment]
> @raelga Yeah but we need rebase to solve the actual problem. As long as the diff includes other commits we cannot ignore this

Sure, I will try to reach @icheikhrouhou over slack to move this PR forward.

/cc @remyleone

**Comment by kbhawkey** [comment]
Hello @icheikhrouhou .
I am closing this pull request since it has been open for some time and needs to be rebased.
You can reopen this PR at any time to continue work.
/close

**Pull request closed without merging**